[feat] allow +server.js files next to +page files (#6773)
sveltejs/kit: documentation/docs/06-form-actions.md

	<!-- content -->
</form>
```

### Alternatives

> Form actions are specifically designed to also work without JavaScript, which means you need to use forms for it. If you don't need the page to work without JavaScript and/or you for example want to interact with an API through JSON, you can instead use `+server.js` files:

Reviewer: @Rich-Harris shared a link somewhere in the forms discussion on Discord about all the ways you might end up without javascript (even if it's not disabled, you might lose a network connection, etc.). I think it'd be helpful to include that here because it had a lot of possibilities I hadn't given much thought to and was really educational and thought provoking

should be "them" instead of "it" since it's referring to "actions" which is plural. the "and/or you for example" portion is a bit tricky to read so here's an alternative for that as well
```suggestion
Form actions are specifically designed to also work without JavaScript, which means you need to use forms with them. If you don't need the page to work without JavaScript, you can instead use `+server.js` files to interact with an API through JSON:
```

Author: Like the rewording, maybe Rich can put the link in here, too, so we can add it (can't find it)

Reviewer 2: To be honest I'm not totally sure this belongs in this section? I think it would be enough to mention in the `+server.js` documentation that it can live alongside `+page` files, and explain the content negotiation rules

Author: I think that's a good thing to do in _addition_ to this paragraph. I saw some people going "oh wait so is this the only way I can do that?" and "so why can't I use this with JSON?". Having this clearly laid out at the bottom of this section is a good thing IMO.
